### ISSUE TITLE
Updated version of "@twilio/conversations": "^2.1.0"

### ISSUE THREAD

**Issue by rizrepos** [TwilioDevEd/conversations-demo#49]
Is there any chance of updating this repo with the latest version of "@twilio/conversations": "^2.1.0" ?

With older version 1, this is working fine but with updated version event listeners are not working, like 

`this.conversationsClient.on("connectionStateChanged"....`

**Comment by nealgist** [comment]
Yes, please update to the latest `@twilio/conversations` package if you can. The demo does not work unless it's the old version.
